main_script.py

```diff
@@ -88,7 +88,21 @@ def concat_dataframe_into_excel(fpath: str):
               "Skipping Step3 and Step4\n")
 
     # Step5 : Scrape country informations
-    df_countries = create_country_table()
+    url_undata = "https://data.un.org/"
```

Comment:
Is it possible for you to include the scrapping from data.un in the function create_country_table(). Like this it simplify the main script for newcomers and it include all step of country creation into the function create_country_table(). 
Would be preferable but if not possible (for time limitation) not a problem :)

Reply:
Remove comments because it's already put into the countries.py file